### DISCUSSION TITLE
Roi矩形选择框

### DISCUSSION THREAD

**Discussion by zhaosn1986** [BYJRK/DotNet-Discussions#71] [MVVM]
您好，看了您很多期Wpf视频，获益匪浅。尤其是行为相关的。记得其中有个拖拽元素的很方便。最近需要用到roi矩形选择框，这方面有没有一些简单好用的？是不是只能手写一个矩形行为啊？

**Comment by BYJRK** [comment]
恐怕你要考虑自己写一个了，可以参考这个：https://stackoverflow.com/questions/1838163/click-and-drag-selection-box-in-wpf

或者也有现成的第三方库？我不太了解

**Reply by zhaosn1986** [reply]
谢谢，很有启发，试试自己写一个吧。

**Reply by spikeOO7** [reply]
第三方库可以用Halcon,可参考这个：https://www.bilibili.com/video/BV1aS4y1W779?p=7&vd_source=424853f7e66505dac665c5670bcc347c
